Makefile

```diff
@@ -59,7 +59,9 @@ pgbouncer_SOURCES = \
 	include/common/unicode_norm.h \
 	include/common/unicode_norm_table.h
 
+UTHASH_HEADER = uthash/src
 pgbouncer_CPPFLAGS = -Iinclude $(CARES_CFLAGS) $(LIBEVENT_CFLAGS) $(TLS_CPPFLAGS)
+pgbouncer_CPPFLAGS += -I$(UTHASH_HEADER)
```

Comment:
You also need to add the uthash license and header file to EXTRA_DIST. Otherwise they won't be part of the resulting tar.gz file that's created by `make dist`. That's also the reason why some CI jobs are failing to build. Because they build from the tarball instead of the git sources.